src/modm/platform/core/hosted/delay.hpp.in

```diff
@@ -74,19 +77,21 @@ delay_ms(int ms)
 inline void
 delayNanoseconds(int)
 {
+	// impossible to do
 	// TODO
 }
```

Comment:
Am I correct to assume you don't have our board implementation included in the build yet, with its timing interface that uses the timing registers and port declarations?

Reply:
correct